exporter/importables.go

```diff
@@ -68,6 +68,10 @@ var (
 		"DBC":        ".dbc",
 		"R_MARKDOWN": ".Rmd",
 	}
+	metastorePrivilegesToAdd = []string{`CREATE_CATALOG`, `CREATE_CONNECTION`,
```

Comment:
only metastore admins can grant these permissions, so if you need to grant yourself these permissions prior to assigning away the ownership

Reply:
this is an interesting case really, maybe we don't need to assign explicit permissions for a metastore... The problem that was reported is that when we create catalogs, schemas, etc., and the owner of an existing metastore is different than the metastore admin who runs the migration code, then we need to have explicit permissions to be able to create these objects...